openslides_backend/action/actions/motion/base_create_forwarded.py

```diff
@@ -0,0 +1,376 @@
+import time
+from collections import defaultdict
+from typing import Any
+
+from openslides_backend.action.actions.motion.mixins import TextHashMixin
+from openslides_backend.shared.typing import HistoryInformation
+
+from ....permissions.permission_helper import has_perm
+from ....permissions.permissions import Permissions
+from ....services.datastore.commands import GetManyRequest
+from ....shared.exceptions import ActionException, PermissionDenied
+from ....shared.filters import FilterOperator
+from ....shared.interfaces.write_request import WriteRequest
+from ....shared.patterns import fqid_from_collection_and_id
+from ...util.typing import ActionData, ActionResultElement, ActionResults
+from .create_base import MotionCreateBase
+
+
+class BaseMotionCreateForwarded(TextHashMixin, MotionCreateBase):
+    """
+    Base create action for forwarded motions.
+    """
+
+    def prefetch(self, action_data: ActionData) -> None:
+        self.datastore.get_many(
+            [
+                GetManyRequest(
+                    "meeting",
+                    list(
+                        {
+                            meeting_id
+                            for instance in action_data
+                            if (meeting_id := instance.get("meeting_id"))
```

Comment:
That is not quite so easy because the prefetch is called before the schema validation.
Not catching this here is going to lead to error messages unintelligible to the user